fe/fe-core/src/test/java/com/starrocks/sql/plan/PruneComplexSubfieldTest.java

```diff
@@ -229,6 +231,70 @@ public void testPruneMapValues() throws Exception {
 
         sql = "select map_keys(map1), map_size(map1) from pc0";
         plan = getVerboseExplain(sql);
-        assertContains(plan, "[/map1/KEY, /map1/OFFSET]");
+        assertContains(plan, "ColumnAccessPath: [/map1/KEY]");
+    }
+
+    @Test
+    public void testPruneComplexFunction() throws Exception {
+        String sql = "select st6.m2[1].s3, st6.a3[2].s6 from sc0";
+        String plan = getVerboseExplain(sql);
+        assertContains(plan, "ColumnAccessPath: [/st6/a3/INDEX/s6, /st6/m2/INDEX/s3]");
```

Comment:
is "[/st6/a3/INDEX/s6" the  a standard representation?

Reply:
None standard representation, just defined by myself